pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -377,6 +377,9 @@ in with passthru; stdenv.mkDerivation {
   '' + optionalString stdenv.isDarwin ''
     # Override the auto-detection in setup.py, which assumes a universal build
     export PYTHON_DECIMAL_WITH_MACHINE=${if stdenv.isAarch64 then "uint128" else "x64"}
+  '' + optionalString (stdenv.isDarwin && x11Support && pythonAtLeast "3.11") ''
+    export TCLTK_LIBS="-L${tcl}/lib -L${tk}/lib -ltcl8.6 -ltk8.6"
```

Comment:
The version number should not be hardcoded.

Reply:
Fixed.